concepts/0441-present-proof-best-practices/README.md

```diff
@@ -0,0 +1,163 @@
+# 0441: Prover and Verifier Best Practices for Proof Presentation
+- Authors: [Stephen Klump](stephen.klump@becker-carroll.com)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-10-31
+- Status Note:  See [RFC 0037](../../features/0037-present-proof/README.md) for the Present Proof protocol.
+- Start Date: 2020-10-31
+- Tags: [concept](/tags.md#concept), [credentials](/tags.md#credentials)
+
+## Summary
+
+This work prescribes best practices for provers in credential selection (toward proof presentation), for verifiers in proof acceptance, and for both regarding non-revocation interval semantics in fulfilment of the Present Proof protocol [RFC0037](../../features/0037-present-proof/README.md). Of particular instance is behaviour against presentation requests and presentations in their various non-revocation interval profiles.
+
+## Motivation
+
+Agents should behave consistently in automatically selecting credentials and proving presentations.
+
+## Tutorial
+
+The subsections below introduce constructs and outline best practices for provers and verifiers.
+
+### Presentation Requests and Non-Revocation Intervals
+
+This section prescribes norms and best practices in formulating and interpreting non-revocation intervals on proof requests.
+
+#### Semantics of Non-Revocation Interval Presence and Absence
+
+The presence of a non-revocation interval applicable to a requested item (see [below](#non-revocation-interval-applicability-to-requested-items)) in a presentation request signifies that the verifier requires proof of non-revocation status of the credential providing that item.
+
+The absence of any non-revocation interval applicable to a requested item signifies that the verifier has no interest in its credential's non-revocation status.
```

Comment:
I suggest adding a clarification along the lines of "A proof may be satisfied by a revocable or non-revocable credential regardless of whether or not a revocation interval information is included in the proof request.  The presence of a revocation interval in a proof request conveys that if the prover is going to use a revocable credential in the proof, a proof of non-revocation must be included. It does not convey any sort of restriction on the revocability of the credential to be proven. In many cases the verifier can not know if the credential to be used by the prover is revocable or not."